docs/README.md

```diff
@@ -34,29 +34,29 @@ macOS (OS X) development environment.
 
 
 # Main features
-
-  * Support for multiple analyzers, currently
-    [_Clang Static Analyzer_](http://clang-analyzer.llvm.org/) and
-    [_Clang-Tidy_](http://clang.llvm.org/extra/clang-tidy/)
-  * Store results of multiple large-scale analysis runs efficiently, either in
-    a PostgreSQL or SQLite database
+## Command line C/C++ Analysis
+  * Executes [_Clang-Tidy_](http://clang.llvm.org/extra/clang-tidy/) and [_Clang Static Analyzer_](http://clang-analyzer.llvm.org/) with Cross-Translation Unit analysis, Statistical Analysis (when checkers are available).
+  * It can create the JSON compilation database
+  * It can automatically analyze gcc cross-compiled projects: detecting gcc compiler configuration and forming the corresponding clang invocations
+  * Incremental analysis
+  * False positive suppression
+  * Result visualization in command line or in static HTML
+
+## Web based report storage
+  * **You can store & visualize thousands of analysis reports** of many analyzers like
+    Clang Static Analyzer, Clang Tidy, Facebook Infer, Clang Sanitizers, Spotbugs (Java), Pylint (python), Eslint (javascript) ...  
```

Comment:
Python, JavaScript

Reply:
@whisperity As a side note you can use the Github [suggestion](https://til.hashrocket.com/posts/szgdi4at59-github-insert-a-suggestion-) feature to mention these bugs.